src/app/components/select-field/index.html

```diff
@@ -164,33 +164,34 @@
   <sky-demo-page-properties sectionHeading="SkySelectFieldPickerContext properties">
     <sky-demo-page-property
       propertyName="data"
-      isOptional="true"
+      isOptional="false"
     >
-      An array of <a stacheRouterLink="." fragment="skyselectfield-properties">The <sky-code>SkySelectField</sky-code> objects</a> provided to the select field's <sky-code>data</sky-code> property. Use this property to populate your custom picker with options for the user to select.
+      Specifies an array <a stacheRouterLink="." fragment="skyselectfield-properties">of <sky-code>SkySelectField</sky-code> objects</a> and provides it to the select field's <sky-code>data</sky-code> property. This populates the custom picker with items for users to select.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="headingText"
       isOptional="true"
     >
-      The text provided to the select field's <sky-code>headingText</sky-code> property. If your custom picker is a modal, use this property for the modal's title.
+      Specifies a header for the select field component's modal and provides it to the select field's <sky-code>pickerHeading</sky-code> property. This property accepts <sky-code>string</sky-code> values.
```

Comment:
The rewording here is also misleading. This value is passed from the select field to the custom picker, not from the consumer to the component. It's up to the author of the custom picker to use this value to set the picker heading if applicable.